src/utils/api/lsk/liskService.js

```diff
@@ -125,6 +125,14 @@ const liskServiceApi = {
     return tabOptions[tab](network, rest);
   },
 
+  getLiskServiceUrl: (networkConfig) => {
+    try {
+      return getServerUrl(networkConfig);
+    } catch (e) {
+      return null;
+    }
+  },
+
```

Comment:
My comment is not limited to this particular method, but documenting inputs and outputs is recommended habit because it helps other developers to know what to pass and what to expect. even for testing, it makes the review process easier.
In addition, can we make sure the output of the function has a unique type. in this case we don't need to perform complex checks when using the output in conditional statements.

Reply:
I added jsDoc to this function. Please check it and let me know what you think.

Reply:
Looks good. Thanks